utils.py

```diff
@@ -0,0 +1,49 @@
+import smtplib
+from email.mime.text import MIMEText
+
+
+CFG_NFO = "/home/{0}/.cfgnfo"
```

Comment:
one thing about the .cfgnfo... it actually is a standard formatted file that can be read with the Python ConfigParser lib.  I'm doing that in the espa-web project... (espa-web/web/ordering/espa_web/settings.py)